_includes/scastie.html

```diff
@@ -14,16 +14,32 @@
 	<div class="call-to-action action-medium">
 		<a href="{{site.data.common.scastieUrl}}" class="button" target="_blank">Learn More</a>
 	</div>
+
 	<div class="wrap">
 		<div class="code-element">
 			<div class="bar-code"></div>
 			<div class="scastie-code-container">
-				<textarea id="scastie-textarea"></textarea>
+        <script src="https://scastie.scala-lang.org/wG96cURqRlmLoOFFa8vHtA.js?theme=dark"></script>
```

Comment:
Really? Why didn’t you pick one of the snippets submitted at the contest?

Reply:
A ops, will do! Which one you think should be it?

Reply:
I like this one: https://scastie.scala-lang.org/dOrqJ7o7Q06dm37Zmb7vKg

Reply:
I get `//snippet not found`